scripts/builder/checker_builder/templates/tokenTargetCalculation.ml.jinja

```diff
@@ -0,0 +1,18 @@
+(* Automatically generated by checker_builder. DO NOT EDIT DIRECTLY! *)
+
+open Common
+open FixedPoint
+
+(** Calculate the current target based on the current quantity, and the current
+    price of kit in ctok (as given by the CFMM).
+    {[
+      target_{i+1} = FLOOR (q_{i+1} / kit_in_ctok_{i+1})
```

Comment:
Is the target still dimensionless if we've removed `index` from this equation? To me this reads: `[1/kit] / [ctok/kit] => [1/ctok]`. Perhaps q has different units in the case of a token-based checker? Not sure if this has implications for the system though.

Reply:
I just re-read the new document covering the units of measure and am still a bit confused about which units `q` is supposed to have. Perhaps we could discuss this offline as well.

Reply:
Yeah, let's discuss this more offline. For reference, here's how I've been thinking about the two cases:
* index-tracking CHF
  ```
  index  : tok/CHF
  p_cfmm : tok/kit
  => (index/p_cfmm) : kit/CHF
  ```
* token-tracking tzCHF
  ```
  p_cfmm : tzCHF/kit
  => (1/p_cfmm) : kit/tzCHF
  ```